java/pom.xml

```diff
@@ -28,6 +28,7 @@
   <url>https://arrow.apache.org/</url>
 
   <properties>
+    <project.build.outputTimestamp>2023-12-13T00:00:00Z</project.build.outputTimestamp>
```

Comment:
The parameter 'project.build.outputTimestamp' is defined at the level of the parent project. Should it also be added to child projects?

Reply:
It's possible but not recommended: if you do artifact:check-buildplan it will raise an error in the case it's inherited from parent pom.

Reply:
Thank you:
````
$ mvn clean artifact:check-buildplan 
[ERROR] project.build.outputTimestamp property should not be inherited but defined in POM /Users/dsusanibar/Documents/business/voltron/fork/arrow/java/format/pom.xml
````

If it is required on all projects, it will give us clarity if it is only defined on child projects and not parent project. What do you think about that?

Reply:
Good point. It makes sense. Let me update the PR accordingly. Thanks !

Reply:
Done, I include the timestamp property only in modules building jar (not pom).